src/utilities/__tests__/findDeprecatedUsages-test.js

```diff
@@ -0,0 +1,84 @@
+/**
+ *  Copyright (c) 2016, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import { findDeprecatedUsages } from '../findDeprecatedUsages';
+import {
+  GraphQLEnumType,
+  GraphQLObjectType,
+  GraphQLSchema,
+  GraphQLString,
+  parse,
+} from '../../';
```

Comment:
I hadn't seen this before. I presume it is equivalent to `'../../index'` (might be better to be explicit then?)